### ISSUE TITLE
0.6.0 broke the "ascii2mathml" binary

### ISSUE THREAD

**Issue by lukasbestle** [runarberg/ascii2mathml#21]
I'm using Node 9.5.0 (latest) and I'm getting the following error with ascii2mathml 0.6.0:

```
$ ascii2mathml something

/usr/local/lib/node_modules/ascii2mathml/index.js:1
(function (exports, require, module, __filename, __dirname) { import parser from "./lib/parser";
                                                              ^^^^^^

SyntaxError: Unexpected token import
    at new Script (vm.js:51:7)
    at createScript (vm.js:138:10)
    at Object.runInThisContext (vm.js:199:10)
    at Module._compile (module.js:624:28)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
```

Seems to be caused by your upgrades to the build process. Version 0.5.4 works just fine.

**Comment by lukasbestle** [comment]
I have rerun the build process locally and now it works. Probably you just need to republish the release.

**Comment by runarberg** [comment]
Sorry for the delayed response @lukasbestle. It should be fixed now and published in 0.6.2.
